### PR TITLE
cleanup in pilot

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
This is minor cleanup.

### CODE REVIEW

File: pilot/pkg/networking/core/v1alpha3/cluster.go [hunk 1]
```diff
@@ -47,11 +47,7 @@ const (
 // Cluster type based on resolution
 // For inbound (sidecar only): Cluster for each inbound endpoint port and for each service port
 func (configgen *ConfigGeneratorImpl) BuildClusters(env *model.Environment, proxy *model.Proxy, push *model.PushContext) ([]*v2.Cluster, error) {
-	clusters := make([]*v2.Cluster, 0)
-
-	services := push.Services
-
-	clusters = append(clusters, configgen.buildOutboundClusters(env, proxy, push, services)...)
```
Comment (rshriram) [inline pilot/pkg/networking/core/v1alpha3/cluster.go]: please undo this change.. this file is changing in another PR and the logic below is also going to change

File: pilot/pkg/networking/core/v1alpha3/cluster.go [hunk 2]
```diff
@@ -92,10 +88,9 @@ func normalizeClusters(clusters []*v2.Cluster) []*v2.Cluster {
 	return out
 }
 
-func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environment, proxy *model.Proxy, push *model.PushContext,
-	services []*model.Service) []*v2.Cluster {
+func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environment, proxy *model.Proxy, push *model.PushContext) []*v2.Cluster {
 	clusters := make([]*v2.Cluster, 0)
-	for _, service := range services {
+	for _, service := range push.Services {
```
Comment (rshriram) [inline pilot/pkg/networking/core/v1alpha3/cluster.go]: and this as well

File: pilot/pkg/networking/core/v1alpha3/cluster.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pilot/pkg/networking/core/v1alpha3/cluster.go]: ok

File: pilot/pkg/networking/core/v1alpha3/cluster.go (hunk 2 above)
Comment (hzxuzhonghu) [reply pilot/pkg/networking/core/v1alpha3/cluster.go]: done

**Comment by hzxuzhonghu** [comment]
@ZackButcher need your re-tag `lgtm`

**Comment by hzxuzhonghu** [comment]
Thanks @rshriram for your so quick response :+1:
